snapcraft/storeapi/errors.py

```diff
@@ -177,7 +177,10 @@ class StoreRegistrationError(StoreError):
     __FMT_RESERVED = (
         'The name {snap_name!r} is reserved.\n\n'
         'If you are the publisher most users expect for '
-        '{snap_name!r} then please claim the name at {register_name_url!r}')
+        '{snap_name!r} then please claim the name at {register_name_url!r}\n\n'
+        'Please register a name that people will associate with you. '
+        'We can rename your snap later if needed.\n\n'
+        'You can push your own versions of firefox, as firefox@nessita.')
```

Comment:
I am interested in where the guidance to write the error this way `'You can push your own versions of firefox, as firefox@nessita.'` came from as I am pretty sure `<snap-name>@<user>` is not a supported form of pushing and a name would need to be registered.

I would use the message from `snapcraft register` as inspiration for this one.

Reply:
It's in the comments by elopio it seems. I was confused as well.

Reply:
Should I remove it or modify it?

Reply:
I would remove it, because it’s needlessly repeating something the user just read before they typed the name (“They would also expect to be able to get other snaps of Thunderbird as ‘thunderbird-nessita’).

If it was retained, two things would need fixing:
- Finding out which punctuation is correct, “-” or “@”
- Changing “nessita” to your actual store account ID.